Forms/MainForm.cs

```diff
@@ -292,12 +305,31 @@ private void tSeed_Leave(object sender, EventArgs e)
             _isUpdating = false;
         }
 
+        public void UpdateSettingString()
+        {
+            try
+            {
+                _settings.Update(tSString.Text);
+                UpdateCheckboxes();
+                ToggleCheckBoxes();
+                tSString.Text = _settings.ToString();
+            }
+            catch
+            {
+                tSString.Text = _oldSettingsString;
+                _settings.Update(_oldSettingsString);
```

Comment:
Shouldn't this also update and toggle checkboxes after rolling back the settings?